morpheushelper/cogs/polls.py

```diff
@@ -41,6 +41,7 @@ async def send_poll(ctx: Context, args: str, field: Optional[Tuple[str, str]] =
         title=question, description=translations.vote_explanation, color=0xFF1010, timestamp=datetime.utcnow()
     )
     embed.set_author(name=str(ctx.author), icon_url=ctx.author.avatar_url)
+    embed.set_footer(text=translations.f_created_by(ctx.author, ctx.author.id), icon_url=ctx.author.avatar_url)
```

Comment:
does it look really nice, one author and footer from the same user in one embed?